### DISCUSSION TITLE
hello!

### DISCUSSION THREAD

**Discussion by cristianvogel** [iPlug2/iPlug2#850] [General]
Just checking in. 

Still wondering if its possible to compile iPlug2 dev flow in https://www.jetbrains.com/clion/ ? 

It uses CMake to build, but makes C++ for beginners that bit easier so I can do more DSP design and less prone to falling into C++ linker errors and other swamps

**Comment by olilarkin** [comment]
I don't know: I don't use Clion, but there is WIP cmake support on this branch https://github.com/iPlug2/iPlug2/tree/linux-2022

**Reply by mccann** [reply]
I made some changes to the cmake support in the Linux branch, for it to support nested cmake builds. However, I'm not specifically targeting Linux. (MacOS) Should I be using linux-2022 instead? or perhaps should I build a patch to bring cmake into the main branch?  Willing to put these into a PR as well, if desired.

**Comment by olilarkin** [comment]
Yes please it would be great to have a PR for the cmake stuff
